Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightPushCommand.swift

```diff
@@ -0,0 +1,252 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+import Foundation
+
+struct TestFlightPushCommand: CommonParsableCommand {
+
+    static var configuration =  CommandConfiguration(
+        commandName: "push",
+        abstract: "Push the local configuration to TestFlight."
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Option(
+        default: "./config/apps",
+        help: "Path to the folder containing the TestFlight configuration."
+    )
+    var inputPath: String
+
+    @Flag(help: "Perform a dry run.")
+    var dryRun: Bool
+
+    func run() throws {
```

Comment:
So the fundamental problem with this command is that you're executing the service requests as you're processing.  This should be more functional.  Eg:

```
compute(localState, remoteState) -> changeSet
execute(changeSet, isDryRun) -> ()
```